packages/twenty-front/src/modules/ui/field/display/components/EllipsisTextWithTooltip.tsx

```diff
@@ -0,0 +1,64 @@
+import { StyledEllipsisDisplay } from '@/ui/field/display/components/EllipsisDisplay';
```

Comment:
looks like this component already exists here `packages/twenty-ui/src/display/tooltip/OverflowingTextWithTooltip.tsx`